test/katex-spec.js

```diff
@@ -1713,6 +1713,19 @@ describe("A MathML font tree-builder", function() {
         expect(markup).toContain("<mo>+</mo>");
     });
 
+    it("should render \\boldsymbol{" + contents + "} with the correct mathvariants", function() {
+        const tex = "\\boldsymbol{" + contents + "}";
+        const tree = getParsed(tex);
+        const markup = buildMathML(tree, tex, defaultOptions).toMarkup();
+        expect(markup).toContain("<mi mathvariant=\"bold-italic\">A</mi>");
+        expect(markup).toContain("<mi mathvariant=\"bold-italic\">x</mi>");
+        expect(markup).toContain("<mn>2</mn>");
+        expect(markup).toContain("<mi mathvariant=\"bold-italic\">\u03c9</mi>");  // \omega
+        expect(markup).toContain("<mi mathvariant=\"bold-italic\">\u03A9</mi>");  // \Omega
+        expect(markup).toContain("<mi>\u0131</mi>");                              // \imath
+        expect(markup).toContain("<mo>+</mo>");
+    });
```

Comment:
We've started using snapshot tests for testing MathML markup, see mathml-spec.js.  Can you add a test there instead?

Reply:
I do not understand what those tests do or how they work. There are no similar tests implemented for \mathbf or any similar command. Could you either provide reference on how to implement that or accept as-is and move to mathml-spec.js later, when \mathbf is moved there too?

Reply:
```
it("should render \\boldsymbol{" + contents + "} with the correct mathvariants", () => {
    const tex = "\\boldsymbol{" + contents + "}";
    const tree = getParsed(tex);
    const markup = buildMathML(tree, tex, defaultOptions).toMarkup();
    expect(markup).toMatchSnapshot();
});
```
You'll need to run the tests once locally in order to update the `katex-spec.js.snap`.  Include the changes to that file in this diff and you should be good.

The tests work by comparing the output of future runs agains the snapshot.  The snapshot can also be regenerated.  I should add more detail to https://github.com/Khan/KaTeX/blob/master/CONTRIBUTING.md#jest-tests along with an `npm` script to regenerate the snapshots when things change.

Reply:
I've added a task for myself: https://github.com/Khan/KaTeX/issues/1013.

Reply:
Done